qiskit/transpiler/preset_passmanagers/level1.py

```diff
@@ -82,10 +82,10 @@ def level_1_pass_manager(pass_manager_config: PassManagerConfig) -> StagedPassMa
         target is not None and target.get_non_global_operation_names(strict_direction=True)
     ):
         pre_optimization = common.generate_pre_op_passmanager(
-            target, coupling_map, remove_reset_in_zero=True
+            target, coupling_map, remove_reset_in_zero=False
         )
     else:
-        pre_optimization = common.generate_pre_op_passmanager(remove_reset_in_zero=True)
+        pre_optimization = common.generate_pre_op_passmanager(remove_reset_in_zero=False)
```

Comment:
Doing this is no-op, with the arguments set like this it returns an empty `PassManager`. It'd probably be more efficient to just default `pre_optimization` to `None` and only set it if we have a coupling map or target available, to run the gate direction passes.

Reply:
I wouldn't have thought that would have an effect on performance, though, and keeping it like this means potentially a bit less coupling between the components?  It doesn't feel like the call site needs/ought to know that this combination of options returns an empty object.

Reply:
It`ll save a few hundred ns at most. It's why I only left a comment and didn't block it because it's probably fine like this. I just wanted to call it out because this whole branch logic was written to support the pre optimization construction that was done prior to the introduction of stages. Where before the optimization loop the passmanagers either ran gate direction and optionally remove reset depending on optimization level or just remove reset in zero if there are no connectivity constraints. That all feels unnecessary now if we're never running remove resets anymore.